src/utils/text.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from 'vitest';
+import { transformToSentenceCase } from './text';
+
+describe('Text transformation utilities', () => {
+  describe('skipTrimToSentenceCase', () => {
```

Comment:
Don't forget to rename the describe block here since we are not using `skipTrimToSentenceCase` anymore! Also since this is for testing, we shouldn't be using the syntax for function names. Tests should ideally be as human-readable as possible as they serve as documentation as well! So for the describe block you can do something like 'trim and transform text to sentence case'